pkg/options/options.go

```diff
@@ -26,11 +26,11 @@ func RunOptions(cmd *cobra.Command, args []string) {
 
 const (
 	// ContainerImageOrg is the org of the default image url
-	ContainerImageOrg = "noobaa"
+	ContainerImageOrg = "alexanderindenbaum"
 	// ContainerImageRepo is the repo of the default image url
 	ContainerImageRepo = "noobaa-core"
 	// ContainerImageTag is the tag of the default image url
-	ContainerImageTag = "master-20230123"
+	ContainerImageTag = "jacky-root-key-rotate"
```

Comment:
remember to update core image once the core PR is merged

Reply:
✅ [here](https://github.com/baum/noobaa-operator/blob/ace19b95668f5acc89288657104b81b57bf041b9/pkg/options/options.go#L28-L33)